plugins/deployment_manager/DeploymentManager.ts

```diff
@@ -388,6 +389,23 @@ export class DeploymentManager {
     }
   }
 
+  tracer(): TraceFn {
+    return (first, ...rest) => {
+      if (typeof first === 'string') {
+        debug(`[${this.network}] ${first}`, ...rest);
+      } else {
+        first.wait().then(async (tx) => {
+          const cost = Number(txCost(tx) / (10n ** 12n)) / 1e6;
+          const logs = tx.events.map(e => `${e.event}(${e.args})`).join(' ');
+          const info = `@ ${tx.transactionHash}[${tx.transactionIndex}]`;
+          const desc = `${info} in blockNumber: ${tx.blockNumber} logs: ${logs}`;
```

Comment:
nit: maybe `events:` instead of `logs:`?

seeing a few `logs: undefined(undefined)` here and there

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/2570291/184298567-456f7e14-4f7e-4171-a072-0991dc778cbe.png">

Reply:
Yeah saw those, that means we don't have the abi, I think maybe still worth knowing?

Reply:
that makes sense, since in this case we're working on a clone of LINK